python-sync-actions/src/component.py

```diff
@@ -288,6 +288,35 @@ def find_array_property_path(response_data: dict, result_arrays: list = None) ->
 
         return result
 
+    def _add_page_params(self, job: Configuration, query_parameters: dict) -> dict:
+        """
+        Add page parameters to the query parameters
+        Args:
+            job: job configuration
+            query_parameters: query parameters
+
+        Returns:
+            query_parameters: updated query parameters
+        """
+
+        if job.api.pagination:
```

Comment:
Takovymu odsazeni
if neco:
--->if neco:
------>
---->asdsad
return query_parameters

se obecne snazime vyhnout. Odsadis tim cely blok jenom a je to potom necitelny, neni jasny kde co konci. Obecne by tam nemely byt vic jak dve urovne vnoreny, jinak je to kandidat na novou funkci nebo restrukturalizaci.

v tomhle pripade je za me lepsi
```
if not job.api.pagination:
   return query_params

....

return query_params
```
kdyz je ten return hned jako prvni prikaz, tak to prehlednosti tolik nevadi.